build.gradle

```diff
@@ -209,7 +209,7 @@ allprojects {
   }
 }
 
-def refTestVersion = 'v0.9.4'
+def refTestVersion = 'v0.10.1' // TODO: Remove this comment (just kicking the CircleCI cache)
```

Comment:
We should probably remove that comment now. :)

Reply:
Done.